### PR TITLE
Add networkpolicy to minio

### PR DESCRIPTION
**Pull request by ozdanborne**
cc: @lachie83

### CODE REVIEW

**Comment by lachie83** [comment]
cc @NitishT

**Comment by lachie83** [comment]
This looks good to me.

**Comment by lachie83** [comment]
Just need a bump of a chart version

File: stable/minio/templates/NOTES.txt [hunk 1]
```diff
@@ -36,3 +36,9 @@ You can now access Minio server on http://<External-IP>:9000. Follow the below s
 
 Alternately, you can use your browser or the Minio SDK to access the server - https://docs.minio.io/categories/17
 {{- end }}
+
+{{ if and (.Values.networkPolicy.enabled) (not .Values.networkPolicy.allowExternal) }}
+Note: Since NetworkPolicy is enabled, only pods with label
+{{ template "fullname" . }}-client=true"
+will be able to connect to this cockroachdb cluster.
```
Comment (nitisht) [inline stable/minio/templates/NOTES.txt]: s/cockroachdb cluster/minio instance

**Comment by nitisht** [comment]
Looks good overall. Thanks for the PR.

**Comment by ozdanborne** [comment]
ping @lachie83 good to go?

**Pull request merged**